l10n_br_sale/report/sale_report_templates.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_saleorder_document_l10n_br_sale" inherit_id="sale.report_saleorder_document">
+        <!-- Substituição do campo usado para Impostos no Brasil -->
+        <xpath expr="//tbody[hasclass('sale_tbody')]//t[@t-as='line']/tr/t[@t-if='not line.display_type']/td[@name='td_taxes']" position="replace">
```

Comment:
Seria interessante trocar o account.tax pelo fiscal.tax caso o company_id.state_id seja igual ao Brasil, se for diferente deveria manter o account.tax. Desta forma mantemos a compatibilidade da impressão do relatório caso a empresa seja de outro país.